processors/blurc/process.c

```diff
@@ -0,0 +1,18 @@
+#include <string.h>
+#include "../../../wasmcv/components/c/wasmcv/imports.h"
```

Comment:
This path should be `#include "wasmcv/imports.h"`and the appropriate CFLAGS set when compiling to find the same directory as mentioned above.

Reply:
shot! ok. will work on that today